src/common/search-engines.ts

```diff
@@ -302,4 +309,52 @@ export const SEARCH_ENGINES: Readonly<Record<SearchEngineId, Readonly<SearchEngi
       name: 'searchEngines_startpageName',
     },
   },
+  searx: {
+    contentScripts: [
+      {
+        matches: [
+          'https://*.searx.be/search',
```

Comment:
1. Searx supports the GET method (you can enable it in the preferences page). An example of GET URL is: https://searx.be/search?q=example&category_general=on&time_range=&language=all . So `https://searx.be/search?*` is also necessary.

2. Would you show an example of `searx.be`'s subdomain? Unnecessary permission for subdomains may lead to potential security problems. For example, if an Searx instance `searx.example` provides login functionality on `login.searx.example`, the users will not want to grant access to passwords that they enter on `login.searx.example` to this extension.
ref: https://developer.chrome.com/docs/extensions/mv3/user_privacy/#required_permissions

Reply:
1. Using GET is a privacy hole. The idea of using Searx is to invisibilize your searches and using "privacy 0" mainstream search engines (Google) with a privacy wall that is searx itself. However GET sends the requested search as a complete URL and your DNS would leak your info with this method. This can be added to the readme file if you want.

2. There was one and I cannot recall which one was, but it was probably one of the instances that doesn't use google so its not being used here. What logging are you talking about?